message-bus-kafka/docker-compose.kafka-utils.yml

```diff
@@ -4,7 +4,7 @@ services:
   kafdrop:
     image: obsidiandynamics/kafdrop:3.27.0
     environment:
-      KAFKA_BROKERCONNECT: 'kafka:9092'
+      KAFKA_BROKERCONNECT: 'kafka-01:9092,kafka-02:9092,kafka-03:9092'
```

Comment:
This should be in the package-metadata.json

Reply:
Done in [2e20304](https://github.com/jembi/platform/pull/219/commits/2e2030445ea1ac3b00df4545286a138255968e41)